news/500-extra-files

```diff
@@ -0,0 +1,24 @@
+Enhancements:
+-------------
```

Comment:
This format may cause a problem when running rever?

Reply:
Oops, yep, totally missed that. Thanks!